### PR TITLE
[1LP][RFR] Introducing other providers for test case in test_relationship.py

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
=================
Introducing other providers for test case related to test_relationship.py

{{pytest: -v -k 'test_host_relationships or test_infra_provider_relationships or test_cloud_provider_relationships' --use-provider complete}}

### CODE REVIEW

**Comment by anurag03** [comment]
Changing the state from WIPTEST to RFR as the test case failed in scvmm2016 due to no host in the provider.

**Comment by quarckster** [comment]
Please rebase.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by anurag03** [comment]
Changed the state from WIPTEST to RFR as their is no value in SCVMM2016 provider and test case are failing for it.

File: cfme/tests/cloud_infra_common/test_relationships.py [hunk 1]
```diff
@@ -139,7 +143,7 @@ def wait_for_relationship_refresh(provider):
 
 @pytest.mark.parametrize("relationship,view", HOST_RELATIONSHIPS,
     ids=[rel[0] for rel in HOST_RELATIONSHIPS])
-@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE)
+@pytest.mark.provider([InfraProvider])
```
Comment (psav) [inline cfme/tests/cloud_infra_common/test_relationships.py]: **REQUIRED:** I think we still want the ONE_PER_TYPE selector in there right? We only want ONE vmware, and ONE rhev etc....

Comment (anurag03) [reply cfme/tests/cloud_infra_common/test_relationships.py]: Okay

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**